packages/dashboard/src/CdcDashboardComponent.tsx

```diff
@@ -516,6 +516,7 @@ export default function CdcDashboard({ initialState, isEditor = false, isDebug =
 
         switch (visualizationConfig.type) {
           case 'chart':
+            visualizationConfig.visualizationType = visualizationConfig.visualizationType === 'Line' && visualizationConfig.formattedData?.length <= 2 ? 'Bar' : visualizationConfig.visualizationType
```

Comment:
I'm not sure why this is necessary. I would think the Chart component could handle this logic itself?